recipes/natsort/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "natsort" %}
+{% set version = "4.0.4" %}
```

Comment:
ok, I'm going to accept this PR, but I'd ask you to update the package once a feedstock for it is made to the latest version. This way you'll have your dependency, but conda-forge won't be out-of-date.